app/src/main/java/de/danoeh/antennapod/fragment/FeedItemlistFragment.java

```diff
@@ -644,6 +645,21 @@ private Feed loadData() {
         return feed;
     }
 
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onKeyUp(KeyEvent event) {
+        switch (event.getKeyCode()) {
+            case KeyEvent.KEYCODE_T:
+                recyclerView.smoothScrollToPosition(0);
+                break;
+            case KeyEvent.KEYCODE_B:
+                recyclerView.smoothScrollToPosition(adapter.getItemCount() - 1);
+                break;
+            default:
+                recyclerView.smoothScrollToPosition(0);
```

Comment:
Please remove the scrolling here. When detecting an unknown key code, it should not scroll.

Reply:
I have removed that line. Thanks.